tests/acceptance.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -Eeo pipefail
+
+expect() {
+  local title="$1"
+  local is="$2"
+  local should="$3"
+
+  echo -e "$title"
+
+  if [[ "$is" != "$should" ]]; then
+     echo -e "    expected(${should})"
+     echo -e "    got(${is})"
+     exit 1
+  fi
+}
+
+case $MODE in
+prepare-only)
+  echo -e "*** testing lsstinstall prepare-only more ***"
+  ./bin/lsstinstall -p -t d_2020_04_13 lsst_distrib
```

Comment:
`d_2020_04_13` may not be kept around on the server forever.  A weekly might be a better choice.